model/system.go

```diff
@@ -170,13 +171,21 @@ var WarnMetricsTable = map[string]WarnMetric{
 		IsBotOnly: false,
 		IsRunOnce: true,
 	},
+	SYSTEM_METRIC_SUPPORT_EMAIL_NOT_CONFIGURED: {
+		Id:         SYSTEM_METRIC_SUPPORT_EMAIL_NOT_CONFIGURED,
+		Limit:      -1,
```

Comment:
Does a `Limit` value of `-1` mean that it'll check every hour for the lifetime of the product?

Reply:
@mkraft We don't use the `Limit` field for this metric. It's just there because its part of the struct.

Reply:
Ok. How often will this check run?

Reply:
It will run each time the server starts, and on top of that once every 7 days.

Reply:
Ah I see, based on `model.WARN_METRIC_JOB_INTERVAL`. Got it, thanks.